### PR TITLE
David

### PR DESCRIPTION
**Pull request by dmcgrouther**
Changed around Navbar.js and Sidebar.js so that when a user clicks in those components they are redirected appropriately and there is not a routing issue.

### CODE REVIEW

**Review by mattlaguardia — APPROVED** [review]
lgtm

**Comment by dmcgrouther** [comment]
two additional fixes:
fixes breadcrumb routing issue
fixes styling issue when looking at a club detail page

File: src/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -49,9 +49,9 @@ const Sidebar = () => {
         <div className="sidebar__card">
         <div className="sidebar__parent">{ current.title }</div>
         { children.map((child, i) =>
-          <Link to={ child.slug } key={ i }>
+          <Link to={ '/' + child.slug } key={ i }>
             <div className="sidebar__link"
-              data-active={!!(pathname === `/${child.slug}`)}>
+              data-active={!!(pathname === `/${child.slug}`) || !!((pathname.split('/').length === 3) && (pathname.split('/')[1] ===child.slug))}>
```
Comment (mattlaguardia) [inline src/components/Sidebar/Sidebar.js:54]: i dont love the `pathname.split` doing this multiple times isnt as performant as doing it once and referencing the output. Could we hoist this logic up, before the return statement, so we can optimize the use? id also like to have you think of a different solution than doing a split method and having to check each property in the array.  there is a more optimized solution. you'll want to do some research into how you could see if a string contains a substring and check against that. 

example: our input is `http://localhost:3000/club/some-club-name' and you'll want to think of a solution that checks to see if the `child.slug` (clubs) exists in the string. You'll want to return a boolean value probably. lmk if you need more explanation and we can pair on this.

**Comment by dmcgrouther** [comment]
code revised for having one link always highlighted for certain pages.

**Review by mattlaguardia — APPROVED** [review]

**Pull request merged**